definitions/artifacts/azure-cognitive-services.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$md": {
+    "access": "public",
+    "name": "azure-cognitive-services"
```

Comment:
One of our guarantees that its impossible to make an "invalid connection" in MD. It looks like SDK impls vary, which means a customer's code would have to change to swap the md _link_ from _Translate_ to _Bing_ ... which means it should be a distinct artifact for each service (despite any similarity in structure).

https://learn.microsoft.com/en-us/python/api/overview/azure/ai-translation-document-readme?view=azure-python&preserve-view=true#translate-your-documents

https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/services/preview/cognitiveservices/v1.0/anomalydetector

Reply:
Renamed the file to `azure-cognitive-service-language.json`